war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -664,18 +664,21 @@ function registerValidator(e) {
   e.targetUrl = function () {
     var url = this.getAttribute("checkUrl");
     var depends = this.getAttribute("checkDependsOn");
+    if (depends === null) {
+      depends = "";
+    }
 
-    if (depends == null) {
+    try {
```

Comment:
is this right? previously it was only eval'ing when depends on wasn't specified?

Won't this eval all the time with how this is setup?

Reply:
Yes, no good way to distinguish between the cases beforehand with the simplified API that I can think of.